src/__tests__/lib/containers/widgets/facet-nav/FacetNav.test.tsx

```diff
@@ -97,7 +97,9 @@ describe('facets display hide/show', () => {
   it('when show more is clicked', async () => {
     //const panel = container.querySelector<HTMLElement>('div.FacetNav')
     const button = container.querySelector('button.FacetNav__showMore')
-    fireEvent.click(button!)
+    act(() => {
```

Comment:
Like this, as it should help test reliability.  Out of curiosity, did you actually encounter a timing issue here (with the async button click)?

Reply:
Actually, I don't think that this change fixed anything 😕. I think `fireEvent` has `act` builtin, so I'm going to remove it. I was trying to reduce the "not wrapped in `act`" warnings but wasn't successful here.